package.json

```diff
@@ -190,7 +190,7 @@
 		"check-engines": "wp-scripts check-engines",
 		"check-licenses": "concurrently \"wp-scripts check-licenses --prod --gpl2\" \"wp-scripts check-licenses --dev\"",
 		"precheck-local-changes": "npm run docs:build",
-		"check-local-changes": "( git diff -U0 | xargs -0 node bin/process-git-diff ) || ( echo \"There are local uncommitted changes after one or both of 'npm install' or 'npm run docs:build'!\" && exit 1 );",
+		"check-local-changes": "( git diff -U0 | xargs -0 node bin/process-git-diff ) || ( echo \"There are local uncommitted changes after one or both of 'npm install' or 'npm run docs:build'!\" && git diff && exit 1 );",
```

Comment:
`bin/process-git-diff` runs `git diff` already:

https://github.com/WordPress/gutenberg/blob/master/bin/process-git-diff.js

There was some issue with `npm` in the past:
https://github.com/npm/npm/issues/17722